xs2a-adapter-service-api/src/main/java/de/adorsys/xs2a/adapter/service/Oauth2Service.java

```diff
@@ -13,6 +13,18 @@
     TokenResponse getToken(Map<String, String> headers, Parameters parameters) throws IOException;
 
     class Parameters {
```

Comment:
what about extracting this class (OauthParameters) and using it in adapters also?

Reply:
Good idea. But as far as I see, now this class has 229 usages among the project, so if you don't mind, I'd propose to leave such a refactoring for some future PR, as this one is huge enough.

Reply:
agree